rqt_py_common/src/rqt_py_common/rqt_roscomm_util.py

```diff
@@ -34,72 +34,76 @@
 
 import os
 
-import genmsg
-import roslaunch
-from roslaunch import RLException
-import rospkg
-import rospy
-import rostopic
+from ament_index_python import get_resources
+
+from rclpy import logging
 
 
 class RqtRoscommUtil(object):
+    _logger = logging.get_logger("RqtRoscommUtil")
 
     @staticmethod
     def load_parameters(config, caller_id):
         """
+        NOTE: Mlautman 11/2/18
+              This function has been deprecated as ROS2 does not yet support setting
+              and getting of parameters in Python
```

Comment:
I am not sure what the idea of this method is but `rclpy` does have support for parameters.

Reply:
I was basing this off the wiki on ros2 that states that bouncy does not have support parameters in python yet. Can you point me to an implementation?

Reply:
Also, this method isn't used anywhere in the RQT packages being ported to ROS2. Is there a reason we wouldn't want to remove it?

Reply:
> I was basing this off the wiki on ros2 that states that bouncy does not have support parameters in python yet.

Python parameters have been added post Bouncy. Since we target the Crystal release we can use them.

> this method isn't used anywhere in the RQT packages being ported to ROS2.

You could follow the history to see when and why it was added.

> Is there a reason we wouldn't want to remove it?

Assuming it was added for a reason and some plugins depend on it if we don't port it now it likely needs to happen some time later.

Reply:
I would like to take care of this in a future PR. As of now, #155 is too big to keep adding features